.github/workflows/build.yml

```diff
@@ -0,0 +1,27 @@
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: build
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ['3.8']
```

Comment:
Why not a more recent version of python, e.g., 3.11?

Reply:
Due to my limited knowledge on Python I am using only very simple and basic features.  The good thing is that it is almost version independent (!)  In any case I take your words and make the changes in the latest commit.